projects/packages/boost-speed-score/src/class-speed-score-request.php

```diff
@@ -102,6 +102,26 @@ public static function generate_cache_id_from_url( $url ) {
 		return hash( self::CACHE_KEY_HASH_ALGO, $url );
 	}
 
+	/**
+	 * Get the list of active modules while this request was created.
+	 *
+	 * @return string
+	 */
+	public function get_active_modules() {
```

Comment:
Can we rename this to `get_active_performance_modules`, given it doesn't just return the active modules any more?